packages/turbo-repository/rust/README.md

```diff
@@ -0,0 +1,43 @@
+## Setup:
+
+Install JS dependencies:
+
+```
+pnpm i
+```
+
+## Build:
+
+Build native library and js wrapper
+
+```
+pnpm exec napi build --platform
+```
+
+## Test
```

Comment:
Do you want to add a simple test case here instead of manual tests? (hint hint this seems like a pretty ideal place to use the node test runner 😸 )

Reply:
At some point, yes. I'd like to end up with some tests to verify many of the example repos we already have.